packages/tests/server/react/formData.test.tsx

```diff
@@ -255,45 +204,47 @@ test('upload a combination of files and non-file text fields', async () => {
   });
 });
 
-test('Throws when aggregate size of uploaded files and non-file text fields exceeds maxBodySize - files too large', async () => {
-  const form = new FormData();
-  form.append(
-    'files',
-    new File(['a'.repeat(50)], 'bob.txt', {
-      type: 'text/plain',
-    }),
-  );
-  form.append(
-    'files',
-    new File(['a'.repeat(51)], 'alice.txt', {
-      type: 'text/plain',
-    }),
-  );
-  form.set('text', 'foo');
-  form.set('json', JSON.stringify({ foo: 'bar' }));
-
-  await expect(
-    ctx.client.uploadFilesOnDiskAndIncludeTextPropertiesToo.mutate(form),
-  ).rejects.toThrowErrorMatchingInlineSnapshot(
-    `[TRPCClientError: Body exceeded upload size of 100 bytes.]`,
-  );
-});
-
-test('Throws when aggregate size of uploaded files and non-file text fields exceeds maxBodySize - text fields too large', async () => {
-  const form = new FormData();
-  form.append(
-    'files',
-    new File(['hi bob'], 'bob.txt', {
-      type: 'text/plain',
-    }),
-  );
-
-  form.set('text', 'a'.repeat(101));
-  form.set('json', JSON.stringify({ foo: 'bar' }));
-
-  await expect(
-    ctx.client.uploadFilesOnDiskAndIncludeTextPropertiesToo.mutate(form),
-  ).rejects.toThrowErrorMatchingInlineSnapshot(
-    `[TRPCClientError: Body exceeded upload size of 100 bytes.]`,
-  );
-});
+// Can bring these back if we want to allow configuring maxBodySize, but for now it's just Infinity and that's okay
```

Comment:
I'm okay

Reply:
Deleted (we can always grab from history)